resources/sts/4.14/openshift_machine_api_aws_cloud_credentials_policy.json

```diff
@@ -1,6 +1,21 @@
 {
   "Version": "2012-10-17",
   "Statement": [
+    {
+      "Sid": "PassRoleToNodes",
+      "Action": [
+        "iam:PassRole"
+      ],
+      "Effect": "Allow",
+      "Resource": "*",
```

Comment:
I updated this PR to relax the resource constraint on MAPI's managed policy due to existing clusters where the `Worker` and `Control-Plane` account roles do not follow the expected naming scheme (see card for more details).

The installer managed policies can continue to be restricted in this way as the permissions aren't relevant during an uninstall of an existing cluster and can be safely updated.